api/README.V1.md

```diff
@@ -147,20 +148,20 @@ This is the data format for both states and counties. `timeseries` is only inclu
   stateName,
   countyName, // null for states
   fips, // 2 digit for states, 5 digit for counties
-  lat, 
+  lat,
   long,
   lastUpdatedDate, // ISO 8601 date string
   actuals: {
     population,
-    intervention, // one of (NO_MITIGATION, MODERATE_MITIGATION, stay_at_home)
+    intervention, // one of (NO_INTERVENTION, MODERATE_INTERVENTION, OBSERVED_INTERVENTION, SELECTED_INTERVENTION)
```

Comment:
Remove SELECTED_INTERVENTION?
Are we missing STRONG_INTERVENTION?

Reply:
SELECTED is still an option here so feels like we should surface it?